### ISSUE TITLE
The Value of PSNR is Nan

### ISSUE THREAD

**Issue by JFcy** [yenchenlin/nerf-pytorch#144]
When I used the mip_Nerf 360 dataset trying to train the model , I get such result  
The Loss and PSNR printed were nan. Is it normal？
![image](https://github.com/user-attachments/assets/6492be71-1106-402a-aeb7-77af582e10a0)
Meanwhile I play the .mp4 file in the responding logs file, it is all dark.
![image](https://github.com/user-attachments/assets/12a999bf-2155-4331-af55-6985339409fd)
the directory structure and the .txt file is :
![image](https://github.com/user-attachments/assets/e643b9c2-92eb-4157-a405-25ca2d42a9ca)
![image](https://github.com/user-attachments/assets/abe13bdc-6062-4c86-a540-fd0adb807e48)

**Comment by PaiDii** [comment]
I met the same problem. Did you solve it?

**Comment by PaiDii** [comment]
I met the same problem. Did you solve it?

**Comment by JFcy** [comment]
> I met the same problem. Did you solve it?

Sorry, I do not solve it. I find it happened when iter achieved 20000.

**Comment by PaiDii** [comment]
I find it happened at 5k iters. And I find another question, like this.
![fine_00001](https://github.com/user-attachments/assets/37ebea6c-d17e-4b51-8c8b-9353d08719a4)
Did you get the same result?
